### PR TITLE
PTX-PTX-15530 Disable validation for metrics-collector for 1.10+

### PR DESCRIPTION
**Pull request by nikolaypopov**
Disable validation for metrics-collector for 1.10+

Signed-off-by: nikolaypopov <nikolay.popov86@gmail.com>

### CODE REVIEW

File: pkg/util/test/util.go [hunk 1]
```diff
@@ -3081,7 +3081,7 @@ func ValidateAlertManagerDisabled(pxImageList map[string]string, cluster *corev1
 func ValidateTelemetryV2Enabled(pxImageList map[string]string, cluster *corev1.StorageCluster, timeout, interval time.Duration) error {
 	logrus.Info("Validate Telemetry components are enabled")
 	opVersion, _ := GetPxOperatorVersion()
```
Comment (pure-jliao) [inline pkg/util/test/util.go:3083]: maybe need to comment this out as well?

File: pkg/util/test/util.go [hunk 2]
```diff
@@ -3537,7 +3537,7 @@ func ValidateTelemetryV2Disabled(cluster *corev1.StorageCluster, timeout, interv
 func ValidateTelemetryV1Enabled(pxImageList map[string]string, cluster *corev1.StorageCluster, timeout, interval time.Duration) error {
 	logrus.Info("Validate Telemetry components are enabled")
 	opVersion, _ := GetPxOperatorVersion()
-	validateMetricsCollector := opVersion.GreaterThanOrEqual(opVer1_11)
+	validateMetricsCollector := opVersion.LessThan(opVer1_10)
```
Comment (pure-jliao) [inline pkg/util/test/util.go:3540]: for v1 there's no difference, so can be consistent with v2.

Comment (nikolaypopov) [reply pkg/util/test/util.go:3540]: We do deploy metrics collector for operators 1.9 or below though

**Review by pure-jliao — APPROVED** [review]

File: pkg/util/test/util.go (hunk 1 above)
Comment (nikolaypopov) [reply pkg/util/test/util.go]: Missed it, removed this line

**Pull request merged**